test/integ/test_put_get_with_gcp_account.py

```diff
@@ -308,12 +343,20 @@ def mocked_file_agent(*args, **kwargs):
     assert original_contents == contents, 'Output is different from the original file'
 
 
-def test_auto_compress_off_gcp(tmpdir, conn_cnx, db_parameters):
+@pytest.mark.parametrize('enable_gcs_downscoped', [False, pytest.param(True, marks=pytest.mark.xfail(
+    reason="Server need to update with merged change. Expected release version: 4.41.0 "))])
+def test_auto_compress_off_gcp(tmpdir, conn_cnx, db_parameters, enable_gcs_downscoped):
     """[gcp] Puts and Gets a small text using gcp with no auto compression."""
```

Comment:
I just realized that this will not allow Jenkins tests to test the new behavior even though they have access to `master` code. We can work around this and mark the tests expected to fail only when running against a PROD deployment by using `is_public_test` fixture. Something like this:
```suggestion
@pytest.mark.parametrize('enable_gcs_downscoped', [True, False])
def test_auto_compress_off_gcp(tmpdir, conn_cnx, db_parameters, is_public_test, enable_gcs_downscoped):
    """[gcp] Puts and Gets a small text using gcp with no auto compression."""
    if enable_gcs_downscoped and is_public_test:
        pytest.xfail("Server need to update with merged change. Expected release version: 4.41.0 ")
```